src/robotFactory.js

```diff
@@ -1,10 +1,83 @@
 'use strict';
 
-class BaseRobot {}
+class BaseRobot {
+  constructor(
+    name,
+    weight,
+    coords,
+    chipVersion
+  ) {
+    this.name = name;
+    this.weight = weight;
 
-class FlyingRobot {}
+    this.coords = {
+      x: coords.x || 0,
+      y: coords.y || 0,
+    };
 
-class DeliveryDrone {}
+    this.chipVersion = chipVersion;
+  }
+
+  getInfo() {
+    return 'Robot: ' + this.name + ', Chip version: '
+      + this.chipVersion + ', Weight: ' + this.weight;
+  }
+
+  goForward(step = 1) {
+    this.coords.y += step;
+  }
+  goBack(step = 1) {
+    this.coords.y -= step;
+  }
+  goRight(step = 1) {
+    this.coords.x += step;
+  }
+  goLeft(step = 1) {
+    this.coords.x -= step;
+  }
```

Comment:
```suggestion
  goForward(step = 1) {
    this.coords.y += step;
  }
  
  goBack(step = 1) {
    this.coords.y -= step;
  }
  
  goRight(step = 1) {
    this.coords.x += step;
  }
  
  goLeft(step = 1) {
    this.coords.x -= step;
  }
```
Add blank line between your methods

Reply:
Ok, ty